.github/containers/Dockerfile

```diff
@@ -0,0 +1,83 @@
+FROM quay.io/centos/centos:stream9 as builder
+
+USER root
+
+# Update and install dependencies
+RUN dnf -y --setopt install_weak_deps=False update && \
+    dnf -y --setopt install_weak_deps=False install java-17-openjdk-devel jss gettext && \
+    dnf clean all
+
+ENV JAVA_HOME=/usr/lib/jvm/jre-17-openjdk
+ENV JRE_HOME=/usr/lib/jvm/jre-17-openjdk
+
+# Copy source code
+COPY . ./candlepin
+WORKDIR /candlepin
+
+# Build the Candlpepin WAR file.
+# Note: we need to includes -Phostedtest=true so that we can run spec tests
+# using our hosted test endpoints.
+RUN ./gradlew war -Phostedtest=true && \
+    mkdir -p /app/build && \
+    cp $(find ./build/libs -name 'candlepin*.war' | head -n 1) /app/build
+
+WORKDIR /app/build
+RUN jar xf $(find . -name 'candlepin*.war' | head -n 1); \
+    sed -i 's/jss4.jar/jss.jar/g' ./META-INF/context.xml
+
+FROM quay.io/centos/centos:stream9
+LABEL author="Josh Albrecht <jalbrech@redhat.com>"
+
+RUN dnf install -y glibc-langpack-en
+ENV LANG en_US.UTF-8
+
+USER root
+
+# Update and install dependencies
+RUN dnf -y --setopt install_weak_deps=False update && \
+    dnf update ca-certificates && \
+    dnf -y --setopt install_weak_deps=False install epel-release java-17-openjdk openssl tomcatjss initscripts wget tar && \
+    dnf clean all
+
+ENV JAVA_HOME=/usr/lib/jvm/jre-17-openjdk
+ENV JRE_HOME=/usr/lib/jvm/jre-17-openjdk
+ENV CATALINA_OPTS=-Djavax.net.ssl.trustStore=$JAVA_HOME/lib/security/cacerts
+
+# Tomcat Setup
+RUN wget https://archive.apache.org/dist/tomcat/tomcat-9/v9.0.76/bin/apache-tomcat-9.0.76.tar.gz; \
```

Comment:
Good question! I meant to add this as a comment, but I forgot to add it. I tried installing Tomcat as a package, but it seems that you need to run Tomcat using systemd when Tomcat is installed using the rpm package. I wanted to avoid using systemd inside of a docker container. From what I understand, it is better to have docker handle the responsibilities of systemd instead of duplicating the responsibilities by running systemd in the container.